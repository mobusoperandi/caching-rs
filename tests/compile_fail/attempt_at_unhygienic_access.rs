use michie::memoized;
use std::collections::HashMap;

#[memoized(key_expr = a, store_type = HashMap<bool, bool>)]
fn f(a: bool) -> bool {
    key
}

fn main() {}
