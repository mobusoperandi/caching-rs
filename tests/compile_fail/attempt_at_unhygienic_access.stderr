error[E0425]: cannot find value `key` in this scope
 --> tests/compile_fail/attempt_at_unhygienic_access.rs:6:5
  |
6 |     key
  |     ^^^ not found in this scope
